src/Compiler/Compiler.cpp

```diff
@@ -1909,6 +1898,14 @@ void Compiler::writeWaveHelp(std::ostream* out, int frontSpacePadCount,
   writeHelp(out, helpEntries, frontSpacePadCount, descColumn);
 }
 
+bool Compiler::HasInternalError() const {
+  if (!m_tclCmdIntegration) {
+    ErrorMessage("Internal: tcl command integration does not initialized");
```

Comment:
Let's no create a separate "Internal" category here. 
```suggestion
    ErrorMessage(TCL command integration did not initialize");
```

Reply:
Done